### PR TITLE
Consolidating Zendesk URLs in to constants file

### PR DESCRIPTION
**Pull request by ryanml**
This further makes use of the Zendesk constants file `ui/helpers/constants/zendesk-url.js` by adding existing hardcoded URLs that were in use codebase-wide.

### CODE REVIEW

File: ui/pages/home/home.component.js [hunk 1]
```diff
@@ -58,10 +59,6 @@ import FlaskHomeFooter from './flask/flask-home-footer.component';
 
 const LEARN_MORE_URL =
```
Comment (ryanml) [inline ui/pages/home/home.component.js:60]: Note: This URL currently 404's, I am not sure of an appropriate substitution if there is any. cc: @bschorchit

**Review by NiranjanaBinoy — DISMISSED** [review]
lgtm

File: ui/helpers/constants/zendesk-url.js [hunk 2]
```diff
@@ -13,6 +13,29 @@ const ZENDESK_URLS = {
...
+    'https://metamask.zendesk.com/hc/en-us/articles/360057142392',
+  USER_GUIDE_CUSTOM_NETWORKS:
+    'https://metamask.zendesk.com/hc/en-us/articles/4404424659995',
+  HARDWARE_CONNECTION:
+    'https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet',
+  BASIC_SAFETY:
+    'https://metamask.zendesk.com/hc/en-us/articles/360015489591-Basic-Safety-Tips',
+  INFURA_BLOCKAGE:
+    'https://metamask.zendesk.com/hc/en-us/articles/360059386712',
+  LEGACY_WEB3: 'https://metamask.zendesk.com/hc/en-us/articles/360053147012',
```
Comment (digiwand) [inline ui/helpers/constants/zendesk-url.js:38]: Thank you for pulling these all together! 

Would be awesome to alphabetize these keys and add this file to our eslint sort rule. Can we alphabetize these here or open a follow-up PR for this? Let me know if you'd like me to open the follow-up PR proposed here as well

Comment (ryanml) [reply ui/helpers/constants/zendesk-url.js:38]: Great suggestion! I'll sort the keys as part of this PR 👍

Comment (ryanml) [reply ui/helpers/constants/zendesk-url.js]: Done!

Comment (digiwand) [reply ui/helpers/constants/zendesk-url.js]: 😄  tysm!

**Review by jpuri — APPROVED** [review]
👍

**Review by NidhiKJha — APPROVED** [review]

**Review by digiwand — APPROVED** [review]
Ty! :D

**Pull request merged**